api/file-translations/params/base-parameters-with-callback.ts

```diff
@@ -0,0 +1,32 @@
+import { BaseParameters } from "../../parameters";
+import { FtsCallback, FtsCallbackMethod, FtsCallbackUserData } from "./fts-callback-parameter";
+
+export class BaseParametersWithCallback extends BaseParameters {
+    setCallback(
+        url: string,
+        httpMethod?: FtsCallbackMethod,
```

Comment:
httpMethod is required (don't know what elvis operator mean hear)

Reply:
it's just an optional parameter. look at method's implementation below, it uses POST as default value if not specified:
```
httpMethod: FtsCallbackMethod = FtsCallbackMethod.POST,
```